recipes/lora_finetune_distributed.py

```diff
@@ -4,6 +4,7 @@
 # This source code is licensed under the BSD-style license found in the
 # LICENSE file in the root directory of this source tree.
```

Comment:
I am updating the file so it's easy to review the UX difference. Do we have preference on adding `if-else` for config `--fsdp2` or adding a new file lora_finetune_distributed_fsdp2.py @karthikprasad  @rohan-varma

Reply:
I think you tagged Karthik by mistake (sorry for the notifications Kartik!)

Yeh I'd vote for a different recipe with its own config instead of if-else blocks. I think BC is important and we shouldn't move off of FSDP until we fully tested this out. Let me know if this makes sense?

Reply:
good catch. I just corrected my tagging

I am good with creating a new recipe